code/Model/SiteTreeLinkTracking.php

```diff
@@ -116,15 +116,21 @@ public function augmentSyncLinkTracking()
         $allFields = DataObject::getSchema()->fieldSpecs($this->owner);
         $linkedPages = [];
         $anyBroken = false;
+        $hasTrackedFields = false;
         foreach ($allFields as $field => $fieldSpec) {
             $fieldObj = $this->owner->dbObject($field);
             if ($fieldObj instanceof DBHTMLText) {
+                $hasTrackedFields = true;
                 // Merge links in this field with global list.
                 $linksInField = $this->trackLinksInField($field, $anyBroken);
                 $linkedPages = array_merge($linkedPages, $linksInField);
             }
         }
 
+        if (!$hasTrackedFields) {
```

Comment:
Please add a comment about why we don't use `$linkedPages` as per https://github.com/silverstripe/silverstripe-assets/pull/558#discussion_r1282653246

Reply:
@lekoala Just pinging you on this one in case you've forgotten about it, given the other almost-identical PR did get merged already.

Reply:
@GuySartorelli done